src/ToDoForm.jsx

```diff
@@ -1,53 +1,115 @@
 import React, { useState } from 'react';
+import { Button, TextField, Select, MenuItem, styled } from '@mui/material';
+import AddIcon from '@mui/icons-material/Add';
+import theme from './Theme';
+
+import ToDo from './ToDo';
 
 function ToDoForm({ onAddTodo }) {
     const [userInput, setUserInput] = useState({
         task: '',
         date: '1',
+        priority: ToDo.priorities.LOW,
     });
 
     const handleChange = (e) => {
-        setUserInput({ task: e.currentTarget.value, date: userInput.date });
+        setUserInput({ ...userInput, task: e.target.value });
```

Comment:
also, what you are doing now is handling a single unified state called userInput.
unified states are generally used for something called a global state (we will get to that as well). simple components states should be simple. let's split the states into 3:
```
const [task, setTask] = useState('');
const [date, setDate] = useState('1'); // btw why string? side task: let's change this to a real date instance (js Date). your date component should be replaced with a datepicker
const [priority, setPriority] = useState(priorities.LOW); // btw why low as default? think about it
```